cwl/src/main/scala/wdl4s/cwl/BaseCommandToCommandParts.scala

```diff
@@ -2,9 +2,10 @@ package wdl4s.cwl
 
 import shapeless.Poly1
 import wdl4s.wdl.command.StringCommandPart
+import wdl4s.wom.CommandPart
 
 object BaseCommandToCommandParts extends Poly1 {
-  implicit def one = at[String] { Seq(_) map StringCommandPart.apply }
+  implicit def one = at[String] { baseCommand => Seq(baseCommand).map(StringCommandPart.apply): Seq[CommandPart] }
```

Comment:
ToL I don't feel strongly about this but am curious why so explicit?

Reply:
I was fighting with compilation errors because the type were not derived properly so I made everything hyper explicit to fix that. It might be too much I'll try to remove this one

Reply:
It might be that IntelliJ can't work it out even if the compiler is happy. I've seen that a few times now

Reply:
Actually it was not compilation error sorry it was runtime failures when running the tests. But I removed it and it worked so this was not the issue :)